src/test/java/com/jcabi/github/mock/MkIssueEventsTest.java

```diff
@@ -0,0 +1,170 @@
+ * permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT
+ * NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND
+ * FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL
+ * THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT,
+ * INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
+ * HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT,
+ * STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED
+ * OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package com.jcabi.github.mock;
+
+import com.jcabi.aspects.Tv;
+import com.jcabi.github.Event;
+import java.util.Date;
+import java.util.Iterator;
+import javax.json.Json;
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.Matchers;
+import org.junit.Test;
+
+/**
+ * Test case for {@link MkIssueEvents}.
+ * @author Chris Rebert (github@chrisrebert.com)
+ * @version $Id$
+ */
+public final class MkIssueEventsTest {
+    /**
+     * MkIssueEvents can create issue events.
+     * @throws Exception If some problem inside
+     */
+    @Test
+    public void createsIssueEvent() throws Exception {
+        final MkIssueEvents events = this.issueEvents();
+        final String login = "jack";
+        final String type = "locked";
+        final long before = now();
+        final Event.Smart event = new Event.Smart(
+            events.create(type, 2, login, null)
+        );
+        final long after = now();
+        MatcherAssert.assertThat(
+            event.type(),
+            Matchers.equalTo(type)
+        );
+        MatcherAssert.assertThat(
+            event.author().login(),
+            Matchers.equalTo(login)
+        );
+        MatcherAssert.assertThat(
+            event.url().toString(),
+            // @checkstyle LineLength (1 line)
+            Matchers.equalTo("https://api.jcabi-github.invalid/repos/jeff/test/issues/events/1")
+        );
+        MatcherAssert.assertThat(
+            event.createdAt().getTime(),
+            Matchers.greaterThanOrEqualTo(before)
+        );
+        MatcherAssert.assertThat(
+            event.createdAt().getTime(),
+            Matchers.lessThanOrEqualTo(after)
+        );
+    }
+
+    /**
+     * MkIssueEvents can create an issue event with a label attribute.
+     * @throws Exception If some problem inside
+     */
+    @Test
+    public void createsIssueEventWithLabel() throws Exception {
+        final MkIssueEvents events = this.issueEvents();
+        final String label = "my label";
+        final Event event = events.create("labeled", 2, "samuel", label);
+        MatcherAssert.assertThat(
+            // @checkstyle MultipleStringLiteralsCheck (1 line)
```

Comment:
@cvrebert we should make the string literals that are used more than once into variables instead of suppressing the rule.